### PR TITLE
Fix support for verify() and error type to fully support verify() in PSA/eBPF backend

### PR DESCRIPTION
**Pull request by tatry**
This PR fixes the support for verify() and error type to fully support them in eBPF backend. Works for PSA and filter architecture.

Co-authored-by: Tomasz Osiński <tomasz@opennetworking.org>
Co-authored-by: Mateusz Kossakowski <mateusz.kossakowski@orange.com>

### CODE REVIEW

File: backends/ebpf/psa/ebpfPsaGen.cpp [hunk 1]
```diff
@@ -27,6 +27,38 @@ limitations under the License.
 
 namespace EBPF {
 
+class PSAErrorCodesGen : public Inspector {
```
[the remark this answers is not in the document]
Comment (tatry) [reply backends/ebpf/psa/ebpfPsaGen.cpp:30]: I use this class only to iterate over error codes and emit them into C code. Similarly is done for [filter architecture](https://github.com/p4lang/p4c/blob/fa63ac4c8790fb30a30d6b9875391b713d3657bb/backends/ebpf/ebpfProgram.cpp#L171-L182).

I've prepared a [PR in my fork](https://github.com/tatry/p4c/pull/5) which shows difference with this midend pass, especially [this commit](https://github.com/tatry/p4c/pull/5/commits/f93db23993a7c1d477141e94c1ffdc6023263f52). In my opinion there is no profits from using  `convertErrors` pass.

[the remark this answers is not in the document]
Comment (mbudiu-vmw) [reply backends/ebpf/psa/ebpfPsaGen.cpp:30]: Sounds fine, it's a small amount of code.

**Comment by mbudiu-vmw** [comment]
Should I merge this or do you plan further changes?

**Comment by mbudiu-vmw** [comment]
I will merge this anyway

**Comment by tatry** [comment]
Thank you. I have no plans for further changes.

**Pull request merged**